.github/workflows/back-tests.yml

```
@@ -42,3 +42,10 @@ jobs:
     - name: Run tests
       working-directory: junqo_back
       run: npm test
+
+  mark-tests-passed-if-no-changes:
+    if: needs.changes-check.outputs.any_changed != 'true'
```

actionlint

property "changes-check" is not defined in object type {}

(expression)